src/App.tsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
-import { ThemeProvider } from 'styled-components';
 import { theme } from './styles/theme';
-import { Router } from './Router';
+import { ThemeProvider } from './styles/styled-components';
+import { AppNavigator } from './Navigator';
+import { AuthProvider, FirebaseProvider } from './core';
 
 type State = { assetsLoaded: boolean };
 
 export default class App extends React.Component<{}, State> {
     state = {
-        assetsLoaded: true,
+        assetsLoaded: true
```

Comment:
I suggest to set prettier to include trailing comma for arrray and object. It has some advantages during code review proccess.

Reply:
I'd prefer too, but there is no such a rule in `.prettierrc` and I didn't make any changes in the code manually.